pages/cathocomponents.js

```diff
@@ -29,9 +24,9 @@ export default function CathoComponentsPage() {
         gtmPrefix="next-header-b2c-nosearch"
         hasSearch
       />
-      <HeaderLoggedResponsive userProfile="STATUS_I_PROFISSIONAL" hasSearch />
```

Comment:
This component  `HeaderLoggedResponsive` still exists in catho components, how about making it come back with it?